packages/start-slicemachine/src/cli.ts

```diff
@@ -0,0 +1,70 @@
+import mri from "mri";
+
+import * as pkg from "../package.json";
+
+type Args = {
+	open: boolean;
+	port: string;
+	help: boolean;
+	version: boolean;
+};
+
+const args = mri<Args>(process.argv.slice(2), {
+	boolean: ["open", "help", "version"],
+	string: ["port"],
+	alias: {
+		port: "p",
+		help: "h",
+		version: "v",
+	},
+	default: {
+		open: false,
+		help: false,
+		version: false,
+	},
+});
+
+if (args.help) {
+	console.info(
+		`
+Usage:
+    start-slicemachine [options...]
+
+Options:
+    --open         Open Slice Machine automatically
+    --port, -p     Specify the port on which to run Slice Machine
+    --help, -h     Show help text
+    --version, -v  Show version
+`.trim(),
+	);
+
+	process.exit();
+}
+
+if (args.version) {
+	console.info(pkg.version);
+
+	process.exit();
+}
+
+let port: number | undefined;
```

Comment:
is there a default for the port?

Reply:
Yes, `start-slicemachine` retains the default `9999` port from today's SM.